pages/apim/3.x/kubernetes/apim-kubernetes-operator-installation-cluster.adoc

```diff
@@ -0,0 +1,199 @@
+[[apim-kubernetes-operator-installation-cluster]]
+= Deploying GKO on a remote cluster
+:page-sidebar: apim_3_x_sidebar
+:page-permalink: apim/3.x/apim_kubernetes_operator_installation_cluster.html
+:page-folder: apim/kubernetes
+:page-layout: apim3x
+
+== Overview
+
+This section provides steps how to deploy the Gravitee Kubernetes Operator (GKO) on an existing remote / cloud-based APIM-ready link:https://k3d.io/[k3d^] Kubernetes cluster.
+
+The deployment involves the following stages:
+
+1. Deploy the GKO on your cluster.
+2. Create a Management Context custom resource.
+3. Create an API Definition custom resource - this creates a new API on the cluster.
+4. Test the new API by calling it through the APIM Gateway.
+
+NOTE: If your architecture requires the management of multiple Kubernetes clusters with a different set of APIs for each cluster, you should deploy the GKO separately on each cluster. Follow the process described below for each cluster deployment.
+
+== Prerequisites
+
+Before you start the deployment process, ensure that you have access to a remote / cloud-based Kubernetes cluster.
+
+It is also recommended to check the current Kubernetes context to ensure that it is set to the correct cluster:
+
+....
+kubectl config current-context
+....
+
+NOTE: See the link:https://kubernetes.io/docs/reference/kubectl/cheatsheet/#kubectl-context-and-configuration[Kubernetes documentation^] for more information on displaying a list of contexts and setting your current context.
+
+== STEP 1: Deploy the GKO on the cluster
```

Comment:
same as my previous msg.
to install GKO on a cluster all these steps are not necessary. only this one install /deploy the operator by the `bundle.yml` way. and int the future we could have other way like OLM and helm 

I'm not sure how to make this clear in the doc. 🤷‍♂️  

perhaps with a reorganization of the sections like :
- Create a local cluster with apim and gw (optional if you already have it)
- Install / deploy gko operator (only bundle.yml way, and cmd to check crd exist by exemple)
- play with it (ManagementContext + your first api definition )

Reply:
All relevant sections fully refactored now.